tests/negative/proxy/http.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package proxy
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+
+	"github.com/coreos/ignition/tests/register"
```

Comment:
`"github.com/coreos/ignition/v2/tests/register"` (and likewise for the one below) should be correct. If you rerun `make vendor`, it should fix this. I think the panic is because its importing the v0.x ignition register function which doesn't know about spec 3.x.